docs/stratis.txt

```diff
@@ -96,9 +96,10 @@ key unset <key_desc>::
      Unset a key in the kernel keyring so it is no longer available for encryption
      operations.
 report <report_name>::
-        Get a report from the daemon regarding its internal state. This command is
-        not supported as the report names and returned JSON schemas are unstable
-        and may be changed at any time.
+        Get a report from the daemon regarding its internal state.
+        Get engine_state_report if no report name is provided. This command only
```

Comment:
What we want here is to indicate which names are stable and which aren't, but making a list of 1 is unappealing.

Maybe say something like:

The `engine_state_report` name will be supported in future releases. Any other report name should be considered unstable and may be removed in a future release. The JSON schema of any report must always be considered unstable.